101-aks-azml-targetcompute/azuredeploy.json

```diff
@@ -0,0 +1,171 @@
+            "metadata": {
+                "description": "A specific IP address and netmask for the Docker bridge, using standard CIDR notation."
+            }
+        },
+        "agentVMSize": {
+            "type": "string",
+            "defaultValue": "Standard_D3_v2",
+            "metadata": {
+                "description": "The size of the VM instances."
+            }
+        },
+        "kuberneteVersion": {
+            "type": "string",
+            "defaultValue": "1.16.9",
+            "metadata": {
+                "description": "Supported Azure Kubernetes version"
+            }
+        },
+        "linuxAdminUsername": {
+            "type": "string",
+            "defaultValue": "azureuser",
+            "metadata": {
+                "description": "User name for the Linux Virtual Machines."
+            }
+        },
+        "sshRSAPublicKey": {
+            "type": "string",
+            "metadata": {
+                "description": "Configure all linux machines with the SSH RSA public key string. Your key should include three parts, for example 'ssh-rsa AAAAB...snip...UcyupgH azureuser@linuxvm'"
+            }
+        }
+    },
+    "variables": {
+		 "dnsPrefix": "[parameters('clusterName')]"
+	},
+    "resources": [
+        {
+            "type": "Microsoft.ContainerService/managedClusters",
+            "apiVersion": "2020-03-01",
+            "name": "[parameters('clusterName')]",
+            "location": "[parameters('location')]",
+            "sku": {
+                "name": "Basic",
+                "tier": "Free"
+            },
+            "identity": {
+                "type": "SystemAssigned"
+            },
+            "properties": {
+                "kubernetesVersion": "[parameters('kuberneteVersion')]",
+                "dnsPrefix": "[variables('dnsPrefix')]",
+                "agentPoolProfiles": [
+                    {
+                        "name": "agentpool",
+                        "count": 3,
+                        "vmSize": "[parameters('agentVMSize')]",
+                        "osDiskSizeGB": "[parameters('osDiskSizeGB')]",
+                        "maxPods": 110,
+                        "type": "VirtualMachineScaleSets",
+                        "orchestratorVersion": "[parameters('kuberneteVersion')]",
+                        "mode": "System",
+                        "osType": "Linux"
+                    }
+                ],
+                "linuxProfile": {
+                    "adminUsername": "[parameters('linuxAdminUsername')]",
+                    "ssh": {
+                        "publicKeys": [
+                            {
+                                "keyData": "[parameters('sshRSAPublicKey')]"
+                            }
+                        ]
+                    }
+                },
+                "servicePrincipalProfile": {
+                    "clientId": "msi",
+                    "secret": null
```

Comment:
remove this property

Reply:
Hi @bmoore-msft  :  Do you mean by : client ID r secret : null ?

Reply:
This is what I promised to use MSI for AKS and it would be useful for customer pick up this. Thanks !!!

Reply:
remove "secret": null